jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/PersistenceUnitParseProcessor.java

```diff
@@ -76,7 +76,7 @@
  */
 public class PersistenceUnitParseProcessor implements DeploymentUnitProcessor {
 
-    private static final String WEB_PERSISTENCE_XML = "WEB-INF/classes/META-INF/persistence.xml";
+    private static final String WEB_PERSISTENCE_XML = "WEB-INF/classes/persistence.xml";
```

Comment:
Per https://jakarta.ee/specifications/persistence/3.1/jakarta-persistence-spec-3.1#_footnotedef_87 the existing code is correct.

Reply:
Ok, I will change this part